### PR TITLE
[Support RP] Updating documentation and examples for 2019-05-01-preview version

### PR DESCRIPTION
**Pull request by shahbj79**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by shahbj79** [comment]
@allenjzhang - Can you please help review? This PR is to update the documentation. There is no change to contracts/APIs.

File: specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json [hunk 1]
```diff
@@ -1439,7 +1447,7 @@
                 },
                 "contactDetails": {
                     "$ref": "#/definitions/UpdateContactProfile",
-                    "description": "Contact details to be updated on the support ticket."
+                    "description": "Contact details to on the support ticket."
```
Comment (allenjzhang) [inline specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json]: nit: I thought the previous statement makes more sense and grammatically correct.

File: specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json [hunk 2]
```diff
@@ -1406,22 +1414,22 @@
...
-                    "description": "The payload of the quota change request.",
+                    "description": "Payload of the quota increase request.",
                     "type": "string"
                 }
             }
 
         },
         "UpdateSupportTicket": {
-            "description": "Updates support ticket.",
+            "description": "Ability to update severity and contact details in the support ticket.",
```
Comment (allenjzhang) [inline specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json]: nit: **Ability** to update ...  feels like describing a boolean flag.  How about: "**Updates to a support ticket**"

**Review by allenjzhang — APPROVED** [review]

File: specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json (hunk 2 above)
Comment (shahbj79) [reply specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json]: Fixed

File: specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json (hunk 1 above)
Comment (shahbj79) [reply specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json]: Good catch, fixed.

**Pull request merged**